### DISCUSSION TITLE
How do I get a bot to talk?

### DISCUSSION THREAD

**Discussion by xNeizy** [PrismarineJS/mineflayer#3126] [Q&A]
I want to play soundtrack with bot help, is it possible to implement this somehow using plasmovoice? The track is in the same file with the script: "music.mp3"

**Comment by amoraschi** [comment]
This plugin should suit your needs: [mineflayer-plasmovoice](https://github.com/Maks-gaming/mineflayer-plasmovoice)

**Reply by xNeizy** [reply]
I tried using this library, took the sample code and inserted it. But when logging on to the server, the bot had headphones with an exclamation mark over his head, which meant that he did not have plasmovoice. I installed the plasmovoice plugin on the local server, it does not work. I tried to install it as a mod, it didn't help either... Code: 

```
const mineflayer = require("mineflayer")
const plasmo = require("mineflayer-plasmovoice")

const bot = mineflayer.createBot({
    "host": "New_2023x8y9.aternos.me",
    "username": "Bot"
})


bot.loadPlugin(plasmo.plugin)

bot.on("voicechat_connected", () => {
    bot.setControlState("sneak", true)

    // Path to file with any audio format
    bot.plasmovoice.SendAudio("music.mp3")
})

bot.on("voicechat_audio_end", () => {
    bot.setControlState("sneak", false)
})
```

**Reply by amoraschi** [reply]
Then you should move this to mineflayer-plasmovoice's repo

**Reply by xNeizy** [reply]
How ?

**Reply by amoraschi** [reply]
Close this one and create an issue there

**Reply by Maks-gaming** [reply]
![image](https://github.com/PrismarineJS/mineflayer/assets/77161824/7ac95717-1572-42cd-a09c-bcc6875dcddb)
nah, he is using aternos

**Reply by amoraschi** [reply]
Please close this @xNeizy
